libs/service-portal/regulations-admin/src/screens/Edit.tsx

```diff
@@ -59,6 +60,7 @@ const Edit = () => {
       draft={draft.data}
       stepName={stepName}
       ministries={ministries.data}
+      userInfo={userInfo}
```

Comment:
Can we not read this from context somehow, inside the `DownloadDraft` component.

Otherwise, I'll add it to the state object, since that is being pushed around anyway.